web/source/keymanweb.js

```diff
@@ -2904,14 +2904,14 @@
     var n,Ln,lgCode;
```

Comment:
I think we need to be very careful that we don't change the registration behaviour with `lgCode`. Looking at the existing code, there were references to both `lgCode` and `PlgCode` and I'm not immediately clear as to the use of each in each case. I think we should define the existing behaviour clearly and then refactor the mess :)